weather.py

```diff
@@ -28,6 +30,8 @@ def convert_date(iso_string):
 
 
 def convert_f_to_c(temp_in_farenheit):
+    temp_in_farenheit = (float(temp_in_farenheit)-32)*(5/9)
+    return round(temp_in_farenheit,1)
```

Comment:
This works, although it's generally good practice to create new variables to store updated values and have them be informative of the value, e.g. `temp_in_celsius` instead of `temp_in_farenheit`. This mainly helps with code readability e.g.
```suggestion
    temp_in_celsius = (float(temp_in_farenheit)-32)*(5/9)
    return round(temp_in_celsius,1)
```